### ISSUE TITLE
unpair a velux bridge

### ISSUE THREAD

**Issue by McMornan** [jlusiardi/homekit_python#214]
Hi, 

the first time I used your lib to pair to my velux gateway worked perfectly. Now my disc has crashed and I haev to setup pairing again. Now it says during discovery that my bridge has already been paired. Is there an easy way to unpair without having access to the old pairing data ?

This is the answer I get during discovery:

Name: VELUX Gateway._hap._tcp.local.
Url: http_impl://192.168.2.131:5001
Configuration number (c#): 13
Feature Flags (ff): Supports HAP Pairing (Flag: 1)
Device ID (id): 11:33:44:55:66:77
Model Name (md): VELUX Gateway
Protocol Version (pv): 1.1
State Number (s#): 1
Status Flags (sf): Accessory has been paired. (Flag: 0)
Category Identifier (ci): Bridge (Id: 2)

When I try to identify it tells me I have insufficient privileges:

 python3 -m homekit.identify -d 11:22:33:44:55:66
identify failed because: Request denied due to insufficient privileges. (-70401).

I then tried sudo ing it and then the answer simply was "step 3" ...

thanks,
Lars

**Comment by jlusiardi** [comment]
Hi Lars,

for your first question: no there is no way to use the unpair command without the pairing data. But there should be an official way to reset the Velux gateway. After that you might be able to pair again.

The second question/issue: `python3 -m homekit.identify -d 11:33:44:55:66:77` is the unpaired (from the devices point of view) version of the identify command. After you have reset the gateway, this should work. If paired, use `python3 -m homekit.identify -f ${PAIRINGDATAFILE} -a ${ALIAS}`.

I hope this helps a little...

Greetings,
Joachim

**Comment by McMornan** [comment]
Well... Thanks for your answer. I managed to salvage the pairing information from my old ssd. With that I could successfully unpair and pair again on the new system.

I also corrected my fatal oversight not to include my pairing information in my daily backup. However it might help others to give the advice to safeguard the pairing information. I do not use Apple and was not aware of that restriction in homekit.

Thanks for your Info, 
Lars
